ana/settings.py

```diff
@@ -119,4 +119,6 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.11/howto/static-files/
 
-STATIC_URL = '/static/'
+STATIC_URL = os.environ.get('STATIC_URL', 'http://localhost:5000/')
```

Comment:
@arnellebalane is there a reason why `/static/` is not the default value? This can be problematic because now the project will not run properly with just `python manage.py runserver` specially the django admin interface

Reply:
@emanlodovice I've undone the changes to the settings variables in ` [9ae148a](/emanlodovice/ana/pull/19/commits/9ae148af5d228b8495e568560235df481a67fc7a)`